scripts/ci/pre_commit/update_installers.py

```diff
@@ -30,8 +30,22 @@
 FILES_TO_UPDATE = [
     AIRFLOW_SOURCES_ROOT_PATH / "Dockerfile",
     AIRFLOW_SOURCES_ROOT_PATH / "Dockerfile.ci",
+    AIRFLOW_SOURCES_ROOT_PATH / "scripts" / "ci" / "install_breeze.sh",
     AIRFLOW_SOURCES_ROOT_PATH / "scripts" / "docker" / "common.sh",
     AIRFLOW_SOURCES_ROOT_PATH / "pyproject.toml",
+    AIRFLOW_SOURCES_ROOT_PATH / "dev" / "breeze" / "src" / "airflow_breeze" / "global_constants.py",
+    AIRFLOW_SOURCES_ROOT_PATH
+    / "dev"
+    / "breeze"
+    / "src"
+    / "airflow_breeze"
+    / "commands"
+    / "release_management_commands.py",
+]
+
+
+DOC_FILES_TO_UPDATE: list[Path] = [
```

Comment:
Will this list be updated? Should we make it a Path instead of `list[Path]`?

Reply:
It's quite likely - maybe we will have other docs like that :)

Reply:
I just wanted to keep it consistent with the "regular" replacement. Even if technically we do not need it now, it is no harm to keep it this way  - otherwise the two snippets of code (for loops below) would look slightly different.

I think consistency is a king in this case.

But happy to hear if you disagree with this assesment of course :D

Reply:
sounds good :) just want to confirm